view/src/components/foodbanks.js

```diff
@@ -230,8 +256,425 @@ class Foodbank extends Component {
+                    <Chip className={classes.chip} label={tag} size="small" />
+                  ))}
+                  <Box
+                    display="flex"
+                    flexDirection="row"
+                    flexWrap="wrap"
+                    padding={0}
+                    margin={0}
+                  >
+                    <Box padding={3}>
+                      <Typography
+                        className={classes.position}
+                        color="textSecondary"
+                      >
+                        Details:
+                      </Typography>
+                      <Typography
+                        variant="body2"
+                        component="p"
+                        className={classes.foodbankLocation}
+                      >
+                        Location of Food Bank: {foodbank.location}
+                        <br />
+                        Refrigeration Space (in pallets):
+                        {foodbank.refrigerationSpaceAvailable}
+                        <br />
+                        Max Load Size (in pallets): {foodbank.maxLoadSize}
+                      </Typography>
+                    </Box>
+                    {/* TODO(andrewhojel): allow user to choose the point of contact! */}
+                    {foodbank.contacts.length > 0 && (
+                      <Box padding={3}>
+                        <Typography
+                          className={classes.position}
+                          color="textSecondary"
+                        >
+                          Point of Contact:
+                        </Typography>
+                        <Typography variant="body2" component="p">
+                          Role: {foodbank.contacts[0]["contactRole"]}
+                          <br />
+                          Name: {foodbank.contacts[0]["contactName"]}
+                          <br />
+                          Phone: {foodbank.contacts[0]["contactPhone"]}
+                          <br />
+                          Email: {foodbank.contacts[0]["contactEmail"]}
+                        </Typography>
+                      </Box>
+                    )}
+                    <Box padding={3}>
+                      <Typography
+                        className={classes.position}
+                        color="textSecondary"
+                      >
+                        Logistics:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Forklift: {foodbank.forklift ? "yes" : "no"}
+                        <br />
+                        Pallet: {foodbank.pallet ? "yes" : "no"}
+                        <br />
+                        Loading Dock: {foodbank.loadingDock ? "yes" : "no"}
+                      </Typography>
+                    </Box>
+                    {/* DETERMINE(andrewhojel): do we include hours of foodbanks? */}
```

Comment:
oy, that's not what I meant when i asked for a change on this. `DETERMINE` isn't...a thing?
```suggestion
                    {/* TODO(andrewhojel): Determine if we should include hours of foodbanks. */}
```

Reply:
(fyi @andrewhojel )